src/appengine/handlers/jobs.py

```diff
@@ -53,6 +53,22 @@ def get_queues():
   return queues
 
 
+def _add_fuzzers_to_job_items(items):
+  """Return a dic of job items along with associated fuzzers."""
+  result_items = []
```

Comment:
Would look a bit nicer implemented like this and avoid the explicit loop:

```python
def _job_to_dict(job):
  result = job.to_dict()
  ...
  return result
```

then...

```python
...
'items': [_job_to_dict(item) for item in items]
...
```

in `get_results`.

Reply:
Should I add the fuzzer addition part in _job_to_dict itself?

Reply:
done